docs/build-windows.md

```diff
@@ -44,7 +47,12 @@ you build a binary optimized for gaming.
 2. Select a **Release** build type in Visual Studio, and run the build.
 
 
-## Build using MSYS2
+## Build using MSYS2 (deprecated)
+
+> **Note**
+>
+> MSYS2 builds are deprecated; we will probably remove support for this in the
+> future.
```

Comment:
See comment above.

Suggest stating:

```
MSYS2 builds are not actively maintained or tested by the development team.

We will probably relocate the MSYS2 notes to a separate markdown file and mention that these builds are currently unmaintained.
```

Reply:
Or if we really want to delete the notes:

```
MSYS2 builds are not actively maintained or tested by the development 
team. We will probably removes these notes in the future.
```

Reply:
Yeah, I'll take this version, thanks. 

We'll just nuke these instructions at some point. I guess I could have done it now, but did not want to do it without @shermp agreeing to it (he created the now disabled MSYS2 workflow).